main.js

```diff
@@ -0,0 +1,18 @@
+//remove 'All Posts' link from the header
+const allPosts = document.querySelector('li');
+allPosts.remove();
+
+//Remove the span element that contains the "Read Time" in the featured post. The featured post is the first post on the page.
+const spanTime = document.querySelector("span");
+spanTime.remove();
+
+// Remove the last post from the page, titled "Stop Planning".
+const stopPlanning = document.querySelector("main .posts article"); 
+stopPlanning.remove();
```

Comment:
So this is passing the test but it not being done correctly. Since querySelector grabs the first article that matches the argument, your code is deleting the first instance of `main .posts article` instead of the final posts as mentioned in the instructions

Reply:
My latest commit reintroduces the solution I previously used; thank you for the great feedback!